### PR TITLE
Fix extra quote on fronts img attrs

### PR DESCRIPTION
**Pull request by sunnyclimber456**
## What does this change?

* Fix the stray `"` on fronts `img` tags, as reported by https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.theguardian.com%2Fus

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## Screenshots

| Before      | After      |
|-------------|------------|
| before | after|

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by dblatcher — APPROVED** [review]

File: common/app/views/fragments/items/elements/facia_cards/image.scala.html [hunk 1]
```diff
@@ -1,11 +1,7 @@
+@import conf.switches.Switches
 @import layout.WidthsByBreakpoint
 @import model.ImageMedia
-@import views.support.{ImgSrc, RenderClasses, SrcSet}
-@import implicits.Requests._
-@import conf.switches.Switches
-@import experiments.{ActiveExperiments}
```
Comment (oliverlloyd) [inline common/app/views/fragments/items/elements/facia_cards/image.scala.html:6]: I presume you removed `implicits.Requests` & `experiments.{ActiveExperiments}` intentionally?

**Review by oliverlloyd — APPROVED** [review]
Looks good, just a question about the imports

File: common/app/views/fragments/items/elements/facia_cards/image.scala.html (hunk 1 above)
Comment (georgeblahblah) [reply common/app/views/fragments/items/elements/facia_cards/image.scala.html:6]: I used IntelliJ's "Optimise Imports" - I think they weren't being used

Comment (georgeblahblah) [reply common/app/views/fragments/items/elements/facia_cards/image.scala.html:6]: I'll add `implicits.Requests` back as I'm not 100% sure about that, but `ActiveExperiments` is no longer referenced in the file

Comment (oliverlloyd) [reply common/app/views/fragments/items/elements/facia_cards/image.scala.html]: Sounds good!

**Pull request merged**